cli/src/semgrep/config_resolver.py

```diff
@@ -307,7 +307,10 @@ def _safe_relative_to(a: Path, b: Path) -> Path:
     @staticmethod
     def _convert_config_id_to_prefix(config_id: str) -> str:
         at_path = Path(config_id)
-        at_path = Config._safe_relative_to(at_path, Path.cwd())
+        try:
+            at_path = Config._safe_relative_to(at_path, Path.cwd())
+        except FileNotFoundError:
+            pass
```

Comment:
Should we just be passing? No need to raise?

Reply:
Not sure why we would raise, this function is completely optional, there's an option to turn of rule name rewriting

Reply:
yeah but if it's turned on might be preferable to fail loudly than have the rules not have the prefix?

Reply:
It seems it's just a sorting/output difference, personally I'd rather semgrep run and have the output be slightly less organized than fail completely

Reply:
I guess that's fair. I was worried about that making it onto app but those rules dont have their path rewritten anyway.